gpjax/__init__.py

```diff
@@ -13,120 +13,37 @@
 # limitations under the License.
 # ==============================================================================
 from gpjax import (
+    base,
     decision_making,
+    gps,
     integrators,
-)
-from gpjax.base import (
-    Module,
```

Comment:
It's probably better to be consistent but `gpjax.Module` (and perhaps the `param_field`) is the one thing I would've just kept in the "top-level" namespace.

Reply:
Resolved in forthcoming commit